app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+    
+  end
```

Comment:
For style purposes, don't leave the blank whitespace in this function. so:
```def index
   end```

or you can write `def index; end` to make it even shorter. (i'm... 97% sure that syntax is correct-- i'm double checking rn)